spec/requests/tags_controller_spec.rb

```diff
@@ -204,6 +204,15 @@
         expect(topic_ids).to_not include(topic_out_of_category.id)
         expect(topic_ids).to_not include(topic_in_category_without_tag.id)
       end
+
+      it "should produce the right next topic URL" do
+        TopicQuery.any_instance.stubs(:per_page_setting).returns(1)
```

Comment:
Can you get rid of the stub by any chances?

Reply:
I can make it a site setting, but it would be used only in this spec.